testing/restricted_kernel_launcher/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "restricted_kernel_launcher"
```

Comment:
The naming seems inconsistent, it should probably be `oak_restricted_kernel_launcher` and not be in the `testing` folder (I have been moving things out of it, there is only one left).

Reply:
I'll happily move it out of the testing dir, but being previously unaware of the context I thought the testing folder was nice place to hide utilities in that aren't otherwise integrated into the rest of our crates.